### PR TITLE
Lethal injection syringes no longer fit into syringe guns

### PR DESCRIPTION
**Pull request by Qwertytoforty**
## What Does This PR Do

Lethal injection syringes no longer fit into syringe guns

## Why It's Good For The Game

Warcrimes must come in small packages 15 unit packages, not 50. This also cuts down on some peoples habits of instantly rushing it on wizard rounds to shoot the wizard, come on, let people have fun.

## Testing

spawned gun and syringe, attempted to load it into syringe gun, failed

## Changelog
:cl:
tweak: Lethal injection syringes no longer fit into syringe guns
/:cl:

### CODE REVIEW

**Comment by GreatGatherer** [comment]
NOOOOO

File: code/modules/projectiles/guns/syringe_gun.dm [hunk 1]
```diff
@@ -68,6 +68,9 @@
 
 /obj/item/gun/syringe/attackby(obj/item/A, mob/user, params, show_msg = TRUE)
 	if(istype(A, /obj/item/reagent_containers/syringe))
+		if(istype(A, /obj/item/reagent_containers/syringe/lethal))
+			to_chat(user, "<span class='warning'>[A] is too big to fit into [src].</span>")
+			return
```
Comment (DGamerL) [inline code/modules/projectiles/guns/syringe_gun.dm:73]: Could do a check of the max size of the syringe instead, for admemery?

Comment (Qwertytoforty) [reply code/modules/projectiles/guns/syringe_gun.dm:73]: If it is admemery, I hope they are being responsible enough to trust them with the ability

**Review by DGamerL — APPROVED** [review]
LGTM

**Review by Burzah — APPROVED** [review]

File: code/modules/projectiles/guns/syringe_gun.dm [hunk 2]
```diff
@@ -68,6 +68,9 @@
 
 /obj/item/gun/syringe/attackby(obj/item/A, mob/user, params, show_msg = TRUE)
 	if(istype(A, /obj/item/reagent_containers/syringe))
+		if(istype(A, /obj/item/reagent_containers/syringe/lethal))
```
Comment (CinnamonSnowball) [inline code/modules/projectiles/guns/syringe_gun.dm:71]: Couldn't you just.. find a beaker and a regular syringe to sidestep this?
Maybe the lethal syringe could be made its own item instead?

**Comment by AdamPrime314** [comment]
### WARNING!! WARNING!! THE CREATOR OF THIS PR DIED TO THE SYRINGE GUN ONCE!!

> This also cuts down on some peoples habits of instantly rushing it on wizard rounds to shoot the wizard

We already have a thing for that, it's called [Battlemage Armor](https://paradisestation.org/wiki/index.php?title=Wizard#Battlemage_Armor). Wanted to make syringes harmless without spending 2 spellbook points anyway just for the meme? Just groom yourself in front of the [mirror](https://paradisestation.org/wiki/index.php?title=Wizard#Magic_Mirror) till you look like you lack a liver. A syringe already got you? Try some [Charcoal](https://paradisestation.org/wiki/index.php?title=Guide_to_Chemistry#Charcoal) from a Toxins first aid kit, it's good for you. I can return to giving you examples in a few hours when I won't be busy. If only it would be possible to [stop time](https://paradisestation.org/wiki/index.php?title=Wizard#Stop_TIme)......

**Comment by TravisAngeI** [comment]
> ### WARNING!! WARNING!! THE CREATOR OF THIS PR DIED TO THE SYRINGE GUN ONCE!!
> > This also cuts down on some peoples habits of instantly rushing it on wizard rounds to shoot the wizard
> 
> We already have a thing for that, it's called [Battlemage Armor](https://paradisestation.org/wiki/index.php?title=Wizard#Battlemage_Armor). Wanted to make syringes harmless without spending 2 spellbook points anyway just for the meme? Just groom yourself in front of the [mirror](https://paradisestation.org/wiki/index.php?title=Wizard#Magic_Mirror) till you look like you lack a liver. A syringe already got you? Try some [Charcoal](https://paradisestation.org/wiki/index.php?title=Guide_to_Chemistry#Charcoal) from a Toxins first aid kit, it's good for you. I can return to giving you examples in a few hours when I won't be busy. If only it would be possible to [stop time](https://paradisestation.org/wiki/index.php?title=Wizard#Stop_TIme)......

i dont think charcoal would drain the chems fast enough to not end up killing you/getting you killed, why should you have to plan ahead for a delayed insta-kill that isn't intended to be used that way?

**Comment by DGamerL** [comment]
> i dont think charcoal would drain the chems fast enough to not end up killing you/getting you killed, why should you have to plan ahead for a delayed insta-kill that isn't intended to be used that way?

I'm 99% sure they just wanted to post something like a copypasta

**Comment by TravisAngeI** [comment]
> > i dont think charcoal would drain the chems fast enough to not end up killing you/getting you killed, why should you have to plan ahead for a delayed insta-kill that isn't intended to be used that way?
> 
> I'm 99% sure they just wanted to post something like a copypasta

A joke could be flying over my head here, but I'm pretty sure they were serious considering they disliked the PR

**Comment by AdamPrime314** [comment]
> dont think charcoal would drain the chems fast enough to not end up killing you/getting you killed

Charcoal drains other chemicals in the blood stream by 1 unit per tick while healing you of 1.5 points of toxin damage, preventing you from succumbing to death as [toxins by default deal 2 points of damage](https://github.com/ParadiseSS13/Paradise/blob/7123f6f05a3a138722f173a21ec826961dfe6ac2/code/modules/reagents/chemistry/reagents/toxins.dm#L12). It will be quick enough for you to wait of the effects in a secure location for 50 ticks (1 minute 40 seconds).

> why should you have to plan ahead for a delayed insta-kill that isn't intended to be used that way?

This isn't a solid argument as the wizard should expect all kinds of intense attacks meant to be as lethal and as quick as possible, while this one is easy to forsee and has easy to come by counters which make codegunning it ridiculous. Given that there are only [**TWO** lethal injection syringes, it's reasonable for it to possess some strength](https://github.com/ParadiseSS13/Paradise/blob/7123f6f05a3a138722f173a21ec826961dfe6ac2/code/game/objects/structures/crates_lockers/closets/secure/security_lockers.dm#L207) over the standard variant, regardless if it was labeled with "**NOTICE: INTENDED FOR USE WITH SYRINGE GUNS**" or not. An unprepared wizard will die to a thought out syringe just as to a lethal injection one packed to the brim with random garbage, trading accessibility for effort.

**Comment by DGamerL** [comment]
> Charcoal drains other chemicals in the blood stream by 1 unit per tick while healing you of 1.5 points of toxin damage, preventing you from succumbing to death as [toxins by default deal 2 points of damage](https://github.com/ParadiseSS13/Paradise/blob/7123f6f05a3a138722f173a21ec826961dfe6ac2/code/modules/reagents/chemistry/reagents/toxins.dm#L12). It will be quick enough for you to wait of the effects in a secure location for 50 ticks (1 minute 40 seconds).

Lethal injection syringes contain ~5 reagents which will deal a variety of damage to you

**Comment by Qwertytoforty** [comment]
This is also mainly because it has 50 volume, instead of 15

**Comment by AdamPrime314** [comment]
> Lethal injection syringes contain ~5 reagents which will deal a variety of damage to you

### Using Charcoal

15 units of Toxin: 0.4 + 1 metabolization rate with 2 toxin damage per tick = 22 toxin damage
10 units of Pancuronium: 0.2 + 1 metabolization rate with 8% chance for a 2 second stun = 0 damage (**UNCRAFTABLE**)
5 units of Cyanide: 0.1 + 1 metabolization rate with 1.5 toxin damage = 7.5 toxin damage
10 units of Fluorosulfuric Acid: 0.4 + 1 metabolization rate with 1 toxin damage per tick and one time 15 fire damage = 8 toxin damage and 15 fire damage
10 units of Fluorine: 0.4 + 1 metabolization rate with 1 toxin damage and 1 fire damage per tick = 8 toxin damage and 8 fire damage
**TOTAL:** 68.5 damage (45.5 toxin damage + 23 fire damage) **THREE SIPS OF CHARCOAL HEAL THE TOXIN DAMAGE**

### Using [Mugwort Tea](https://github.com/ParadiseSS13/Paradise/blob/7123f6f05a3a138722f173a21ec826961dfe6ac2/code/modules/reagents/chemistry/reagents/food_reagents.dm#L606) (Wizard starter free healing)

15 units of Toxin: 0.4 + 5 metabolization rate with 2 toxin damage per tick = 6 toxin damage
10 units of Pancuronium: 0.2 + 5 metabolization rate (Does nothing due to not reaching enough ticks) = 0 damage (**UNCRAFTABLE**)
5 units of Cyanide: 0.1 + 5 metabolization rate with 1.5 toxin damage = 1.5 toxin damage
10 units of Fluorosulfuric Acid: 0.4 + 5 metabolization rate with 1 toxin damage per tick and one time 15 fire damage = 2 toxin damage and 15 fire damage
10 units of Fluorine: 0.4 + 5 metabolization rate with 1 toxin damage and 1 fire damage per tick = 2 toxin damage and 2 fire damage
**TOTAL:** 28.5 damage (11,5 toxin damage + 17 fire damage) **THREE SIPS OF MUGWORT TEA HEALS THIS**

**Comment by TravisAngeI** [comment]
> > Lethal injection syringes contain ~5 reagents which will deal a variety of damage to you
> 
> ### Using Charcoal
> 15 units of Toxin: 0.4 + 1 metabolization rate with 2 toxin damage per tick = 22 toxin damage 10 units of Pancuronium: 0.2 + 1 metabolization rate with 8% chance for a 2 second stun = 0 damage (**UNCRAFTABLE**) 5 units of Cyanide: 0.1 + 1 metabolization rate with 1.5 toxin damage = 7.5 toxin damage 10 units of Fluorosulfuric Acid: 0.4 + 1 metabolization rate with 1 toxin damage per tick and one time 15 fire damage = 8 toxin damage and 15 fire damage 10 units of Fluorine: 0.4 + 1 metabolization rate with 1 toxin damage and 1 fire damage per tick = 8 toxin damage and 8 fire damage **TOTAL:** 68.5 damage (45.5 toxin damage + 23 fire damage) **THREE SIPS OF CHARCOAL HEAL THE TOXIN DAMAGE**
> 
> ### Using [Mugwort Tea](https://github.com/ParadiseSS13/Paradise/blob/7123f6f05a3a138722f173a21ec826961dfe6ac2/code/modules/reagents/chemistry/reagents/food_reagents.dm#L606) (Wizard starter free healing)
> 15 units of Toxin: 0.4 + 5 metabolization rate with 2 toxin damage per tick = 6 toxin damage 10 units of Pancuronium: 0.2 + 5 metabolization rate (Does nothing due to not reaching enough ticks) = 0 damage (**UNCRAFTABLE**) 5 units of Cyanide: 0.1 + 5 metabolization rate with 1.5 toxin damage = 1.5 toxin damage 10 units of Fluorosulfuric Acid: 0.4 + 5 metabolization rate with 1 toxin damage per tick and one time 15 fire damage = 2 toxin damage and 15 fire damage 10 units of Fluorine: 0.4 + 5 metabolization rate with 1 toxin damage and 1 fire damage per tick = 2 toxin damage and 2 fire damage **TOTAL:** 28.5 damage (11,5 toxin damage + 17 fire damage) **THREE SIPS OF MUGWORT TEA HEALS THIS**

Okay, lemme just say this.
Using lethal injection syringe is a cheesy tactic that takes minimal effort to do, and it ruins the fun for the wizard player. So therefore this is a good PR.

**Comment by AdamPrime314** [comment]
> Using lethal injection syringe is a cheesy tactic that takes minimal effort to do, and it ruins the fun for the wizard player. So therefore this is a good PR.

It is in fact "low to nothing" effort level due to the only thing preventing you from using it is not knowing lethal injections syringes exist, but there is only two of them at your supply which obviously balances it out as I mentioned before. Trading availability for effort required in making your own actual mix. I provided you with a list of hard counters to it and calculated right in front of you that it's in no way lethal even with simple Charcoal, and deals laughable damage when using Mugwort that the Wizard is already expected to use, yet you use no arguments to show it's being overused or too simple.

**Comment by TravisAngeI** [comment]
> > Using lethal injection syringe is a cheesy tactic that takes minimal effort to do, and it ruins the fun for the wizard player. So therefore this is a good PR.
> 
> It is in fact "low to nothing" effort level due to the only thing preventing you from using it is not knowing lethal injections syringes exist, but there is only two of them at your supply which obviously balances it out as I mentioned before. Trading availability for effort required in making your own actual mix. I provided you with a list of hard counters to it and calculated right in front of you that it's in no way lethal even with simple Charcoal, and deals laughable damage when using Mugwort that the Wizard is already expected to use, yet you use no arguments to show it's being overused or too simple.

I'm just sayin, it's preferrable if the wizard doesn't have to do any of this and can just play the round normally and get beaten to death

Figure I'll ask this in case you reply, why are you actively hating on this PR?

**Review by S34NW — APPROVED** [review]

**Pull request merged**